### PR TITLE
Change envoy to istio/envoy release-1.5.

### PR DESCRIPTION
**Pull request by jplevyak**

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -38,17 +38,17 @@ bind(
 # 2. Update .bazelversion, envoy.bazelrc and .bazelrc if needed.
 #
 # envoy-wasm commit date: Jan 23 2020
-ENVOY_SHA = "59be1b34f15d23821d8ad22c75e6ad8086334398"
+ENVOY_SHA = "277a86a6aaf2f4a22f9060969fb0cca"
```
Comment (dgn) [inline WORKSPACE]: I guess this should be `277a86a6aaf2f4a22f9060969fb0cca4ab8d7a79`?

**Comment by PiotrSikora** [comment]
Could you update the SHA to pickup the TSan fix?

**Comment by PiotrSikora** [comment]
@jplevyak per my comment above, could you pick the updated SHA instead?

Also, `/retest`ing won't help, since the required `wasmsdk` version is missing:
```
+ ENVOY_SHA=277a86a6aaf2f4a22f9060969fb0cca4ab8d7a79
+ IMAGE=gcr.io/istio-testing/wasmsdk
+ TAG=277a86a6aaf2f4a22f9060969fb0cca4ab8d7a79
+ docker pull gcr.io/istio-testing/wasmsdk:277a86a6aaf2f4a22f9060969fb0cca4ab8d7a79
Error response from daemon: manifest for gcr.io/istio-testing/wasmsdk:277a86a6aaf2f4a22f9060969fb0cca4ab8d7a79 not found: manifest unknown: Failed to fetch "277a86a6aaf2f4a22f9060969fb0cca4ab8d7a79" from request "/v2/istio-testing/wasmsdk/manifests/277a86a6aaf2f4a22f9060969fb0cca4ab8d7a79".
+ echo 'gcr.io/istio-testing/wasmsdk:277a86a6aaf2f4a22f9060969fb0cca4ab8d7a79 does not exist'
```

File: WORKSPACE [hunk 2]
```diff
@@ -38,17 +38,21 @@ bind(
 # 2. Update .bazelversion, envoy.bazelrc and .bazelrc if needed.
 #
 # envoy-wasm commit date: Jan 23 2020
```
Comment (gargnupur) [inline WORKSPACE]: change this to istio/envoy?

File: WORKSPACE (hunk 1 above)
Comment (jplevyak) [reply WORKSPACE]: Fixed.

File: WORKSPACE (hunk 2 above)
Comment (jplevyak) [reply WORKSPACE]: The commit date is for the envoy-wasm fork into istio/envoy after that it is just cherry-picks, so I was going to leave it, but perhaps istio/envoy would be better if less informative in some ways. WDYT?

**Comment by kyessenov** [comment]
@fpesce Please approve. We need to remap proxy to use istio/envoy instead of envoy-wasm.

File: WORKSPACE (hunk 2 above)
Comment (gargnupur) [reply WORKSPACE]: yeah otherwise,  wasm here seems very confusing..

**Comment by PiotrSikora** [comment]
@fpesce FWIW, this should have been done as part of cutting the branch. Maybe add it to the checklist?

@jplevyak could you please bump the SHA to `e85398d7876a71a17475ee634e0599a70df2a0ea`?

**Pull request merged**